.mergify.yml

```diff
@@ -21,6 +22,28 @@ pull_request_rules:
           - s:review-needed
       queue:
         merge_method: squash
+  - name: Auto merge (feature branch)
+    conditions:
+      - base!=main
+      - "#approved-reviews-by>=1"
+      - "#changes-requested-reviews-by=0"
+      - check-success=Coverage
+      - check-success=Format
+      - check-success=Checks (clippy)
+      - check-success=Checks (parachain)
+      - check-success=Checks (standalone)
+      - check-success=Quick check benchmarks
+      - check-success=Test standalone build
+      - check-success=Test parachain build
+      - check-success=Fuzz
+      - label=s:accepted
+    actions:
+      label:
+        remove:
+          - s:in-progress
+          - s:review-needed
+      queue:
+        merge_method: squash
```

Comment:
Looks fine to me, although I'm wondering if we should really squash when merging into feature branches.

Reply:
I think squashing is fine. It will show a reference to the PR primarily and as additional logs all the commits are preserved, so after all no data is lost and in fact it improves readability.